### PR TITLE
Give different result from Resultset::delete() depending on success

### PR DESCRIPTION
**Pull request by 1RV34**
Hello!

* Type: bug fix / new feature (unsure, it's unwanted behavior that is inconsistent with the behavior of `Model::delete()`)
* Link to issue: #11133

**In raising this pull request, I confirm the following (please check boxes):**

- [x] I have read and understood the [Contributing Guidelines](https://github.com/phalcon/cphalcon/blob/master/CONTRIBUTING.md)?
- [x] I have checked that another pull request for this purpose does not exist.
- [ ] I wrote some tests for this PR.

Small description of change: When any `Model::delete()` fails the result bubbles to cause `ResultSet::delete()` to return `false` as well.

Thanks

### CODE REVIEW

**Comment by Jurigag** [comment]
Update changelog also and add some simple test.

**Review by sergeyklay — APPROVED** [review]

**Review by Jurigag — CHANGES_REQUESTED** [review]
Update changelog, add test

**Comment by 1RV34** [comment]
I've added a message to the changelog but not sure if it's according to the standards that those messages should be.

Secondly, I'm not completely sure how to properly create a test for this. Basically if any of the `Model::delete()` calls that are made fails and causes a transaction rollback to happen the result should be `false` just like the `Model::delete()` result. Could I get some help with this?

File: CHANGELOG.md [hunk 1]
```diff
@@ -22,6 +22,7 @@
 - Fixed `Phalcon\Mvc\Micro:handle` to correctly handle `before` handlers [#10931](https://github.com/phalcon/cphalcon/pull/10931)
 - Fixed `Phalcon\Mvc\Micro:handle` to correctly handle `afterBinding` handlers
 - Added way to disable setters in `Phalcon\Mvc\Model::assign` by using `Phalcon\Mvc\Model::setup` or ini option
+- Made `Resultset::delete()` return `false` when any of the `Model::delete` calls fails [PR #12828](https://github.com/phalcon/cphalcon/pull/12828) fixing [issue #11133](https://github.com/phalcon/cphalcon/issues/11133)
```
Comment (Jurigag) [inline CHANGELOG.md]: No need for link ot pr, just only add issue like other changes aboves. Also without "issue"

**Comment by Jurigag** [comment]
About test - just return some random false in some of model::delete, then check if result returned by resultset->delete is false.

**Comment by sergeyklay** [comment]
Just updated CHANGELOG

**Review by sergeyklay — APPROVED** [review]

**Comment by sergeyklay** [comment]
Thank you

**Pull request merged**